octobot_trading/exchanges/implementations/exchange_simulator.py

```diff
@@ -156,8 +164,9 @@ def get_fees(self, symbol=None):
     #     'rate': percentage, // the fee rate, 0.05% = 0.0005, 1% = 0.01, ...
     #     'cost': feePaid, // the fee cost (amount * fee rate)
     # }
-    def get_trade_fee(self, symbol, order_type, quantity, price,
-                      taker_or_maker=ExchangeConstantsMarketPropertyColumns.TAKER.value):
+    def get_trade_fee(self, symbol, order_type, quantity, price, taker_or_maker):
```

Comment:
Is this not breaking tests ?

Reply:
Exchange test are passing. I tested tentacles tests before this change, I'll have a look at tentacles tests